app/components/participant-creator/component.js

```diff
@@ -32,13 +32,20 @@ export default Ember.Component.extend({
     _generate(batchSize, tag) {
         var ret = [];
         for (let i = 0; i < batchSize; i++) {
-            ret.push(`${makeId(5)}${tag ? `-${tag}` : ''}`);
+            var generate = true;
+            while (generate) {
+                var id = `${makeId(5)}${tag ? `-${tag}` : ''}`;
+                if (!ret.contains(id) || !this.get('store').peekRecord('account', id)) {
```

Comment:
`peekRecord` only gets things that have been loaded from the server into the store during this page load. (unlike `findRecord`, it doesn't fire a network request)

How do we ensure that an account with this ID was not created on a previous run of the experiment?